runtime/src/protocol.rs

```diff
@@ -17,6 +17,7 @@ use thiserror::Error;
 
 use crate::{
     common::{cbor, logger::get_logger, namespace::Namespace, version::Version},
+    consensus::tendermint::BACKEND_NAME,
```

Comment:
Maybe only import `tendermint` and refer to it as `tendermint::BACKEND_NAME` below. This makes it more clear which backend this is.